.travis.yml

```diff
@@ -88,7 +88,7 @@ jobs:
       python: 3.4
     - stage: tests
       script: bash $RUN_SCRIPT
-      env: WEBFRAMEWORK=django-master
+      env: WEBFRAMEWORK=django-2.0
```

Comment:
Did you remove testing `django-master` with `python-3.4` on purpose?  (just asking because for 3.5 and 3.6 you added it in addition..)

Reply:
yes, Django 2.1 will only support Python 3.5+, and master already fails on 3.4